python/manylinux1/build_arrow.sh

```diff
@@ -79,7 +79,6 @@ mkdir -p "${ARROW_BUILD_DIR}"
 pushd "${ARROW_BUILD_DIR}"
 cmake -DCMAKE_BUILD_TYPE=Release \
     -DARROW_DEPENDENCY_SOURCE="SYSTEM" \
-    -DZLIB_ROOT=/usr/local \
```

Comment:
Wasn't it necessary?

Reply:
Seems like it is not https://travis-ci.org/ursa-labs/crossbow/jobs/559242639#L809